dll/win32/imm32/imm.c

```diff
@@ -850,6 +850,9 @@ BOOL WINAPI ImmDestroyContext(HIMC hIMC)
  */
 BOOL WINAPI ImmDisableIME(DWORD idThread)
 {
+#ifdef __REACTOS__
+    return NtUserDisableThreadIme(idThread); /* confirmed */
```

Comment:
The 'confirmed' comment is not needed in my opinion.

Reply:
Deleted the comment.